api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -117,10 +117,16 @@ message EndpointLoadMetricStats {
 // Per cluster load stats. Envoy reports these stats a management server in a
 // :ref:`LoadStatsRequest<envoy_api_msg_load_stats.LoadStatsRequest>`
 // [#not-implemented-hide:] Not configuration. TBD how to doc proto APIs.
+// Next ID: 7
 message ClusterStats {
   // The name of the cluster.
   string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
 
+  // The eds_cluster_config service_name of the cluster.
+  // It's possible that two clusters send the same service_name to EDS,
+  // in that case, the management server is supposed to do aggregation on the load reports.
+  string cluster_eds_service_name = 6;
```

Comment:
Hmm, my original comment was lost I think. In any case I think this should just be `service_name`, for consistency with the EDS declaration in existing CDS. I.e. `cluster_service_name`.

Reply:
heh, I erased the previous commit by a "force push" , which I hope was just "folding local new commits" into a "new commit".
I don't have strong feeling about the name, my idea is the eds_cluster_service_name tells where this name "comes from" (cluster::eds_cluster_config::service_name), whereas any a single service_name may not convey that notion.

Updated.